pandas/conftest.py

```diff
@@ -473,6 +473,15 @@ def _create_series(index):
 }
 
 
+@pytest.fixture
+def period_series():
```

Comment:
@WillAyd did we settle on a naming scheme for these?

Reply:
Seems reasonable; we already have a datetime_series so this follows that pattern